docs/quickstart.rst

```diff
@@ -35,8 +35,8 @@ just add some extra initialization code:
         config = Configurator(settings=settings)
 
         cliquet.initialize(config, __version__)
-
-        return config.make_wsgi_app()
+        app = config.make_wsgi_app()
+        return cliquet.install_middlewares(app)
```

Comment:
Question: don't we want to leave the wrapping only in the monitoring section ?

I fear one thing : the first impression of cliquet might be : "this is too much initialization magic", "too far from vanilla pyramid"

If one day we have general purpose middleware we will add it here, in the general introduction docs.

Thoughts ?